resources/test_data/tbl/meta_tables/meta_segments.tbl

```diff
@@ -0,0 +1,9 @@
+table|chunk_id|column_id|column_name|column_type|encoding
```

Comment:
`data_type` instead of `column_type`?

Reply:
I like prefixing all the (duplicated) column information accordingly.

Reply:
`column_data_type`?